flink-queryable-state/flink-queryable-state-java/pom.xml

```diff
@@ -110,8 +110,8 @@ under the License.
 		</dependency>
 
 		<dependency>
-			<groupId>com.data-artisans</groupId>
-			<artifactId>flakka-testkit_${scala.binary.version}</artifactId>
+			<groupId>com.typesafe.akka</groupId>
```

Comment:
I could not find any use of akka testkit in the queryable state client (and there should not be, design wise).

Was this just copy/paste forwarded and can be dropped?

Reply:
There was a test in there that was using Akka but I think that's not needed anymore. Will remove.

On a side node, currently, the queryable-state-client module contains both the client and server implementations (that's why there was the akka test) while flunk-runtime only has interfaces. Fully tweezing the client and server apart is still in progress.